azure-pipelines.yml

```diff
@@ -262,6 +262,14 @@ stages:
           artifact: RuleDescriptorBin
           targetPath: analyzers\src\SonarAnalyzer.RuleDescriptorGenerator\bin
 
+      - powershell: .\scripts\build\set-azp-project-version-variable.ps1
+
+      - task: Cache@1
```

Comment:
Is it version 1 on purpose? Did you try to play with v2? I don't know what the difference is.

Reply:
Yes, I've tried again with version 2 and it failed.